maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -705,47 +705,47 @@ def _on_empty_return(self, event: AtomEvent):
         port.on_consignee -= payload.quantity
         port.empty += payload.quantity
 
-    def _on_action_received(self, event: CascadeEvent):
+    def _on_action_received(self, event: ActualEvent):
```

Comment:
the event for action is a CascadeEvent, never be an AtomEvent

Reply:
Fixed. Also for CitiBike.